src/main/resources/assets/admin/common/js/app/wizard/WizardHeaderWithDisplayNameAndName.ts

```diff
@@ -205,8 +209,8 @@ export class WizardHeaderWithDisplayNameAndName
             !this.ignoreGenerateStatusForName && this.isNameEmptyOrEqualDisplayName(this.displayNameEl.getValue()));
     }
 
-    setName(value: string) {
-        this.nameEl.setValue(value);
+    setName(value: string, silent?: boolean) {
```

Comment:
Codacy complains about missing return type on this one.

Reply:
Fixed